scripts/remove

```diff
@@ -28,8 +27,5 @@ sudo rm -f /etc/systemd/system/$app.service
 # remove database!!!
 sudo mongo rocketchat --eval "db.dropDatabase()"
 
-# remove user!!!
-sudo userdel -r $serviceuser
```

Comment:
no because restore has the same logic as install (create user only if not exist) see [here](https://github.com/YunoHost-Apps/rocketchat_ynh/pull/15/files#diff-68321f788a23b70a33d17888e31bf3efR42)